definitions/pipelines/rnaseq.cwl

```diff
@@ -101,13 +101,7 @@ steps:
         in:
             kallisto_index: kallisto_index
             strand: strand
-            fastqs:
-                source: bam_to_trimmed_fastq_and_hisat_alignments/fastqs
-                valueFrom: |
-                    ${
-                      for(var i=0;i<self.length;i++){self[i] = self[i].reverse()}
-                      return(self)
-                     }
+            fastqs: bam_to_trimmed_fastq_and_hisat_alignments/fastqs
```

Comment:
Since the ordering can be problematic (apparently) depending on which engine is used, maybe we should switch `bam_to_trimmed_fastq_and_hisat_alignments.cwl` to use the specified read1 and read2 FASTQ outputs that the trimming step already provides?  Combining them into the right array of arrays here might still involve a little work, but it'd always be right!

Reply:
I agree, that'd be preferable, but I've got no time to figure that out at the moment - will file it away for a future date :)